### PR TITLE
Login and Signup page

### PR DESCRIPTION
**Pull request by tushartiwari7**
- Login page has
    - form heading
    - email input
    - password input
    - login btn
    - create account btn

- Signup page has
    - heading
    - firstname
    - lastname
    - email
    - password
    - confirm password
    - checkbox to accept T&C
    - Create Account btn
    - login page redirection 
    
   
login
signup

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -5,8 +5,66 @@
...
+               class="input full-width px-sm py-xs my-xs rounded-s"
+         />
+      </div>
+      <div class="m-xs">
+        <label class="fs-s">
+          Password 
+        </label>
+        <input 
+               type="password" 
+               placeholder="********"
```
Comment (paarthbajaj) [inline pages/login.html]: Change the placeholder for password input

File: pages/login.html [hunk 2]
```diff
@@ -5,8 +5,66 @@
...
+    <form class="card p-sm">
+      <div class="h3  text-center fw-semibold m-xs">LOGIN</div>
+      <div class="m-xs">
+        <label class="fs-s">
+          Email 
+        </label>
+        <input 
+               type="email" 
+               placeholder="tushar@furnhouse.com"
+               class="input full-width px-sm py-xs my-xs rounded-s"
```
Comment (paarthbajaj) [inline pages/login.html:40]: You should add a required keyword, your login page's and signup page's input elements are not mandatory. Make them mandatory.

**Review by paarthbajaj — COMMENTED** [review]
Please make the requested changes

**Review by harshalini — APPROVED** [review]
Code is good, you can merge it

**Review by paarthbajaj — APPROVED** [review]
Looks good now.

**Pull request merged**